src/components/Modals/CreateSingleItemModal/CreateSingleItemModal.tsx

```diff
@@ -945,7 +945,7 @@ export default class CreateSingleItemModal extends React.PureComponent<Props, St
     return (
       <EditThumbnailStep
         isLoading={!!isLoading}
-        blob={contents ? toWearableWithBlobs({ contents, isEmote: true }) : undefined}
+        blob={contents ? toEmoteWithBlobs({ contents }) : undefined}
```

Comment:
This should be 
```suggestion
        blob={blob}
```
right?

Reply:
There’s no ‘blob’ variable in that scope 👀

Reply:
sorry, GitHub trolled me 😛 
![image](https://user-images.githubusercontent.com/8763687/191039508-3d6cc0a9-f104-4764-b90a-79aaf9955e5c.png)
Now that I'm seeing it correctly, we only need the `toEmoteWithBlobs` here cause it's only for emotes 👍

Reply:
hey @juanmahidalgo!  the method renderWearablePreview is used in the import file step to generate the initial thumbnail for emotes and wearables.